plugins/linode/personal_access_token.go

```diff
@@ -31,9 +32,32 @@ func PersonalAccessToken() schema.CredentialType {
 		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
 		Importer: importer.TryAll(
 			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryConfigFile(),
 		)}
 }
 
 var defaultEnvVarMapping = map[string]sdk.FieldName{
 	"LINODE_CLI_TOKEN": fieldname.Token,
 }
+
+// TryConfigFile looks for the token in the ~/.config/linode-cli file.
+func TryConfigFile() sdk.Importer {
+	return importer.TryFile("~/.config/linode-cli", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		configFile, err := contents.ToINI()
+		if err != nil {
+			out.AddError(err)
+			return
+		}
+
+		fields := make(map[sdk.FieldName]string)
+		for _, section := range configFile.Sections() {
+			if section.HasKey("token") && section.Key("token").Value() != "" {
+				fields[fieldname.Token] = section.Key("token").Value()
+			}
+		}
+
+		out.AddCandidate(sdk.ImportCandidate{
```

Comment:
Consider adding a `NameHint` with the username (`acme` in your example). This will append the username to the name suggested to users to use for the 1Password item that will include the imported item. This will be useful for users that have more than one Linode access tokens.

Reply:
Regarding multiple credentials, I personally think this is a lesser-used thing, since one creential can work across an entire account, and multiple services can therefore be using the same credential. At least, in my expeience, I use one account that has access to many different services. I'm also not too clear on exactly where to pull the name hint from, as the account name is not quite the same kind of field in the config file as the token.